basic/exercise7/script7.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+function calculateDaysUntilToNextChristmas() {
+
+    var oneDay = 1000 * 60 * 60 * 24;
+    var today = new Date;
+    var Christmas = new Date(today.getFullYear(), 11, 25);
+    if (today.getMonth() == 11 && today.getDate() > 25) {
```

Comment:
You use 11 and 25 as magic numbers in several different places. Maybe it should be specified and referenced as variable? It would allow you to easily change this function to calculate number of days to any date.